### DISCUSSION TITLE
Adding a package doesn't seem to add a line to pyproject.toml

### DISCUSSION THREAD

**Discussion by songololo** [pdm-project/pdm#1058] [Q&A]
Sometimes when adding a package PDM will successful install the package and add it to the lock file, but the package doesn't get added to `pyproject.toml`.

Is there something I'm maybe misunderstanding or doing wrongly?

For example, I've just installed per `pdm add jupyter --dev` and everything runs successfully, gets added to `pdm.lock`, but nothing in `pyproject.toml`.

**Comment by frostming** [comment]
Firstly, make sure you are looking at the correct `pyproject.toml` file, by typing `pdm info`
Secondly, it would be helpful to show the previous state of `pyprojec.toml`, to reproduce this issue. (At least I cannot)

**Reply by songololo** [reply]
I'm not sure why this happens, I wonder if it isn't more of an IDE issue where the memory and disk-based versions of files fall out of sync. For me it seems to be working fine again but if it happens again, and if I can figure out how to reproduce, I'll post a followup.

Thanks for the package!
